examples/resources/awscc_apigateway_request_validator/apigateway_request_validator.tf

```diff
@@ -0,0 +1,11 @@
+resource "awscc_apigateway_rest_api" "MyDemoAPI" {
+  name        = "MyDemoAPI"
+  description = "This is my API for demonstration purposes"
+}
+
+resource "awscc_apigateway_request_validator" "example" {
+  name                        = "example"
+  rest_api_id                 = awscc_apigateway_rest_api.MyDemoAPI.id
```

Comment:
just suggestion
did you want to keep it lower case for the rest api id name?

Reply:
Yes. I kept it as aws provider naming convention.